src/wombats/game/decisions/smoke.clj

```diff
@@ -0,0 +1,89 @@
+(ns wombats.game.decisions.smoke
+  (:require [wombats.game.decisions.helpers :as dh]
+            [wombats.game.utils :as gu]
+            [wombats.arena.utils :as au]))
+
+(defn- get-smoked-coords
+  "Return a collection of coords that should be smokescreened given a central starting point"
+  [coords arena-dims]
+  (conj (map #(gu/adjust-coords coords % arena-dims)
+             (range 8))
+        coords))
+
+(defn- calculate-throw-direction
+  "determines the direction smoke should be thrown based off of the
+  decision makers orientation and the specified throw direction"
+  [decision-maker-orientation throw-direction]
+  (condp = throw-direction
```

Comment:
~Is there an advantage of using `condp` over `case`?~

Actually, I think it might be better to use `case` b/c it's not a sequential search to find the match, but instead it's a constant time match. 

> Unlike cond and condp, case does a constant-time dispatch, the
clauses are not considered sequentially.

[Source](https://clojuredocs.org/clojure.core/case)